.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
+}
```

Comment:
Please, do not add gitignore and IDE settings to the commit, especially in PRs.

Reply:
ye my bad thought i removed it